zanata-war/src/main/java/org/zanata/action/DashboardAction.java

```diff
@@ -59,7 +60,8 @@
 
 @Name("dashboardAction")
 @Scope(ScopeType.PAGE)
-@Restrict("#{identity.loggedIn}")
+@ZanataSecured
+@CheckLoggedIn
```

Comment:
>>    * Is there a sanity test to make sure that classes which need @ZanataSecured actually have it?

> No unfortunately. We could use scannotation or some other tools to scan the entire package on classpath, and check to see any of the class that has security annotations. But the cost may be too high considering our classpath. Unless you want to use external tools like ```find zanata-war/src/main/java/ -type f -name "*.java" | xargs grep "@CheckLoggedIn"``` and ```find zanata-war/src/main/java/ -type f -name "*.java" | xargs grep "@ZanataSecured"```

I think we should have such a test.  Otherwise it's too easy to think we've secured a class when we haven't. Scannotation (or Reflections) only needs to check `org.zanata.*` packages.  I'm sure we have tests which run slower, to test less critical things.